Practicum/Pract. 03/zad2.1.cpp

```diff
@@ -0,0 +1,158 @@
+#include <iostream>
+#include <fstream>
+
+const unsigned MAX_SIZE = 100;
+constexpr char FIRST_MATRIX_FILE[] = "Matrix1.txt";
+constexpr char SECOND_MATRIX_FILE[] = "Matrix2.txt";
+constexpr char SUM_FILE[] = "SumMatrix.txt";
+
+struct Matrix
+{
+	int matrix[MAX_SIZE][MAX_SIZE];
+	int rows = 0;
+	int cols = 0;
+};
+
+void readMatrix(Matrix& Rmatrix,std::ifstream& in)
+{
+	char delimiter;
+	int value;
+	int row = 0;
+	while (row < MAX_SIZE && in)
+	{
+		int col = 0;
+		while (col < MAX_SIZE && in >> value)
+		{
+			Rmatrix.matrix[row][col++] = value;
+			in.get(delimiter);
+
+			if (delimiter == '|')
+			{
+				break;
+			}
+		}
```

Comment:
try using stringstream for this situation (ss is great for spliting).
Also extract a function